### ISSUE TITLE
Toonkor source needs an update to base URL

### ISSUE THREAD

**Issue by juleast** [tachiyomiorg/tachiyomi-extensions#18744]
### Source information

Toonkor 1.4.4

### Source language

Korean

### Source new URL

https://toonkor302.com

### Other details

I realize that these sort of sources change their domains bi-weekly or monthly, but the base URL is completely unrelated to the current URL so it can't redirect or do anything. The current base URL is `tkor.dog` which is no longer a valid domain for the source and redirects to a blank page.

### Acknowledgements

- [X] I have updated all installed extensions.
- [X] I have opened WebView and checked that the source URL is not updated yet.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by MikeZeDev** [comment]
Maybe fetch the link from their Telegram? https://t.me/s/new_toonkor

**Comment by alessandrojean** [comment]
No as using Telegram API in this case will be tricky.

**Comment by juleast** [comment]
Well as of right now, the base of the URL isn't changing so couldn't an automatic domain number change be implemented like on the Newtoki (KR) extension? So just leave `https://toonkor` as a constant and you just interpolate the constant string and the changing domain number. Something like `url = '${base}${domain_num}.com'`

**Comment by MikeZeDev** [comment]
> No as using Telegram API in this case will be tricky.

Dont really need the API if we can read the page properly.

![image](https://github.com/tachiyomiorg/tachiyomi-extensions/assets/10671518/460a3344-5ca6-40bb-adcc-fcd58afd0695)

I'd bet i will work with a fetch, CSS selector and regex to check if message matches toonkor\d+.com .
Again i dont know how Tachi usually proceed.

**Comment by alessandrojean** [comment]
This doesn't seem like a good and solid solution anyway.

**Comment by juleast** [comment]
Or there could be both. Changing the base of the URL and the domain number updating automatically. Maybe this is too overkill of an implementation for one extension lol.

**Comment by MikeZeDev** [comment]
> This doesn't seem like a good and solid solution anyway. Having a baseUrl preference would be better.

Exactly. If one exists ill gladly use it too :) Sorry for bothering you.
